site/content/docs/11-deploying.md

```diff
@@ -61,4 +61,100 @@ When deploying to [Now][], you can pass the environment variable into Now itself
 now -e SAPPER_TIMESTAMP=$(date +%s%3N)
 ```
 
-[Now]: https://zeit.co/now
\ No newline at end of file
+[Now]: https://zeit.co/now
+
+
+## Deploying to App Engine
+
+[Google App Engine](https://cloud.google.com/appengine/) is a fully managed serverless platform. The app engine will run the dynamic server so its not necessary to perform `npm run export`. Before deploying you need to add an `app.yaml` file to your project, that specifies the required node runtime:
+
+```
+# app.yaml
+runtime: nodejs10
+```
+
+A [custom build step](https://cloud.google.com/appengine/docs/standard/nodejs/running-custom-build-step) can be run by adding `"gcp-build": "run-p build"` as a `scripts` entry to your `package.json` scripts. This ensures that the build step will be performed before the application starts. Now you can deploy using the `gcloud app deploy` command.
+
+
+## Deploying to Docker
```

Comment:
nit: I might say "Deploying with Docker".  E.g. you're deploying to Google Cloud Run with Docker

Reply:
Do you really want an update? Maybe its better not to add this, because cloud providers are always changing things and it may become obsolete and will generate more questions to answer. Probably better to focus on building code instead...

Reply:
I imagine the Docker part should stay relatively constant, but if you want to drop the Google Cloud stuff from this PR that's fine with me

Reply:
Can do a new PR, because i might have removed my repo when i was cleaning up.